src/components/UserIcon/UserIcon.module.css

```diff
@@ -0,0 +1,23 @@
+.userIcon {
+  display: block;
```

Comment:
.userIcon should have `postion: relative` so the ::before element is positioned according to the position of the element with .userIcon class